content/en/tracing/guide/init_resource_calc.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Init container resource usage
+disable_toc: false
+---
+
+### Overview
+
+Starting in Agent [v7.60+][1], Datadog uses dynamic resource calculation for init containers that inject tracing libraries. Instead of fixed values, the init containers temporarily request all available CPU and memory for the pod, without affecting how the pod is scheduled. (Prior v7.60, init containers used conservative defaults: `50m` for CPU and `20Mi` for memory.)
+
+This behavior improves tracer startup reliability while respecting Kubernetes scheduling rules. Init containers run sequentially and exit before application containers start, so they don't compete for runtime resources.
```

Comment:
```suggestion
This behavior improves tracer startup reliability while respecting Kubernetes scheduling rules. Since init containers run sequentially and exit before application containers start, they don't compete for runtime resources.
```